GeoTIFF_Standard/standard/clause_4_terms_and_definitions.adoc

```diff
@@ -239,21 +216,29 @@ usually rectangular pattern of parallel scanning lines forming or corresponding
 
  [Source: ISO 19123:2005, 4.1.30]
 
+NOTE: In the the 1995 GeoTIFF v1.0, "A continuous planar space in which pixel values are
+visually realized."
+
 === rational <TIFF>
 
-a RATIONAL value is a fractional value represented by the ratio of two unsigned 4-byte integers
+a _rational_ value is a fractional value represented by the ratio of two unsigned 4-byte integers
 
 === rectified grid
+__georectified grid__
```

Comment:
rectified grid / georectified grid are not mentionned elsewhere than in this section. To be deleted ? orthorectified is mentionned in annex-b as can be seen below and we have an entry for it already

```
$ grep -r "rectified" .
./clause_4_terms_and_definitions.adoc:=== georectified grid
./clause_4_terms_and_definitions.adoc:__rectified grid__
./clause_4_terms_and_definitions.adoc:Note 1 to entry:	If the coordinate reference system is related to the earth by a datum, the grid is a georectified grid.
./clause_4_terms_and_definitions.adoc:=== orthorectified grid
./clause_4_terms_and_definitions.adoc:georectified grid created using ground control points and elevation data where constant scale is maintained throughout the grid
./annex-b.adoc:      Case 5: The raster data cannot be fit into the model space with a simple affine transformation (rubber-sheeting required). Use only the ModelTiepoint tag, and specify as many tiepoints as your application requires. Note, however, that this is not a Baseline GeoTIFF implementation, and should not be used for interchange; it is recommended that the image be geometrically rectified first, and put into a standard projected coordinate reference system.
./annex-b.adoc:We have an aerial photo which has been orthorectified and resampled to a UTM grid, zone 60, using WGS 84 coordinate reference system; the coordinates of the upper-left corner of the image is are given in easting/northing, as 350807.4m, 5316081.3m. The scanned map pixel scale is 100 meters/pixels (the actual dpi scanning ratio is irrelevant).
./annex-b.adoc:===== Unrectified Aerial photo, known tiepoints, in degrees.
```